pom.xml

```diff
@@ -33,7 +33,8 @@
 	<modules>
 		<module>spring-ai-alibaba-core</module>
 		<module>spring-ai-alibaba-starter</module>
-	</modules>
+        <module>spring-ai-alibaba-autoconfig</module>
```

Comment:
I think it's better to use `autoconfigure`, for both the module name and package name.

Reply:
oh!,I found the problem and have recommitted to fix it